doc/whatsnew/2/2.15/index.rst

```diff
@@ -29,6 +29,50 @@ Marc Byrne became a maintainer, welcome to the team !
 
 .. towncrier release notes start
 
+
+What's new in Pylint 2.15.1?
+----------------------------
+Release date: 2022-09-06
+
+- Fix ``used-before-assignment`` for functions/classes defined in type checking guard.
+
+  Closes #7368 (`#7368 <https://github.com/PyCQA/pylint/issues/7368>`_)
+- Update ``modified_iterating`` checker to fix a crash with ``for`` loops on empty list.
+
+  Closes #7380 (`#7380 <https://github.com/PyCQA/pylint/issues/7380>`_)
+- The ``docparams`` extension now considers typing in Numpy style docstrings
+  as "documentation" for the ``missing-param-doc`` message.
+
+  Refs #7398 (`#7398 <https://github.com/PyCQA/pylint/issues/7398>`_)
+- Fix false positive for ``unused-variable`` and ``unused-import`` when a name is only used in a string literal type annotation.
+
+  Closes #3299 (`#3299 <https://github.com/PyCQA/pylint/issues/3299>`_)
+- Fix false positive for ``too-many-function-args`` when a function call is assigned to a class attribute inside the class where the function is defined.
+
+  Closes #6592 (`#6592 <https://github.com/PyCQA/pylint/issues/6592>`_)
+- Fix ``used-before-assignment`` for functions/classes defined in type checking guard.
+
+  Closes #7368 (`#7368 <https://github.com/PyCQA/pylint/issues/7368>`_)
+- Fix ignored files being linted when passed on stdin.
+
+  Closes #4354 (`#4354 <https://github.com/PyCQA/pylint/issues/4354>`_)
+- ``missing-return-doc``, ``missing-raises-doc`` and ``missing-yields-doc`` now respect
```

Comment:
Doesn't `towncrier` know about white spaces?  😅

Reply:
I manually removed the duplicated titles, which is why there might be some problems.

Reply:
Let's fix in #7426